### DISCUSSION TITLE
Publicly available verification of nekRS

### DISCUSSION THREAD

**Discussion by mattfalcone1997** [Nek5000/nekRS#608] [General]
I have read in a recent [preprint](https://arxiv.org/abs/2409.06404v1) that nekRS has been validated against nek5000 and through the method of manufactured solutions. I was wondering if there are any papers or reports that document this. I am also interested in whether proof of order-of accuracy for the spatial and temporal discretisation has been documented anywhere.

**Comment by stgeke** [comment]
This has not been documented in a publicly available publication. However, both the spatial and temporal discretization have been verified. Additionally, the code includes various CI tests to validate it against analytical solutions.

**Reply by mattfalcone1997** [reply]
Okay thanks, I will have a look at the CI tests!

**Reply by stgeke** [reply]
I suggest to check the ethier example. It's our basic NS validation case.
